azure-pipelines/steps/build_linux.yml

```diff
@@ -7,13 +7,19 @@ parameters:
 
 steps:
   - bash: |
+      sudo add-apt-repository ppa:ubuntu-toolchain-r/test
       sudo apt-get update
+      sudo apt install gcc g++
+      sudo apt install gcc-8 g++-8
       sudo apt-get install -y cmake ninja-build libcppunit-dev libgtk-3-dev libpoppler-glib-dev portaudio19-dev libsndfile-dev liblua5.3-dev libzip-dev
+      g++ --version
     displayName: 'Install dependencies'
   - bash: |
       mkdir build
     displayName: 'Create build directory'
   - bash: |
+      export CXX=g++-8
+      export CC=gcc-8
```

Comment:
We will have to add this requirement to our manual build instructions.

Reply:
This has to be done at the time we merge.